### PR TITLE
Use TimeLimit from os-autoinst-common to get OPENQA_TEST_TIMEOUT_DISABLE

### PR DESCRIPTION
**Pull request by Martchus**

### CODE REVIEW

File: docs/Contributing.asciidoc [hunk 1]
```diff
@@ -211,7 +211,7 @@ it when running tests. Example:
 
 [source,sh]
 ----
-TEST_PG='DBI:Pg:dbname=openqa_test;host=/dev/shm/tpg' OPENQA_BASEDIR= prove -v t/14-grutasks.t
+TEST_PG='DBI:Pg:dbname=openqa_test;host=/dev/shm/tpg' OPENQA_BASEDIR= prove -Iexternal/os-autoinst-common/lib -v t/14-grutasks.t
```
Comment (okurz) [inline docs/Contributing.asciidoc:214]: I would really appreciate if we can find a better approach. I do not want to call prove with this additional argument always and also I do not want to tell our developers everytime what's wrong

**Review by okurz — CHANGES_REQUESTED** [review]

File: docs/Contributing.asciidoc (hunk 1 above)
Comment (perlpunk) [reply docs/Contributing.asciidoc:214]: Maybe we can add a symlink?
Or add a `use lib ...` in every test, like we already do with `use lib "$FindBin::Bin/lib";`

**Review by perlpunk — APPROVED** [review]

File: external/os-autoinst-common/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This repository is to be used as a
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/git-subrepo.svg)](https://repology.org/project/git-subrepo/versions)
 
-and in `devel:openQA`, e.g. https://download.opensuse.org/repositories/devel:/openQA/openSUSE_Leap_15.1
+and in `devel:openQA`, e.g. https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2/
```
Comment (okurz) [inline external/os-autoinst-common/README.md:11]: but devel:openQA is not the same as devel:openQA:Leap:15.2 . The latter is correct but then I guess the first should say `devel:openQA:Leap:15.2` as well, right?

Comment (Martchus) [reply external/os-autoinst-common/README.md:11]: Regardless what the answer to your question is: This change needed to go to os-autoinst-common first and then pulled in again. You can ask the creator of the [original PR](https://github.com/os-autoinst/os-autoinst-common/pull/2) what he was thinking at the time :-)

Comment (okurz) [reply external/os-autoinst-common/README.md:11]: Likely the answer is always "not much thinking" :D ok, this is just an update from an external repo. I am still struggling with this sub repo approach when it also causes confusions like this :D

**Review by okurz — CHANGES_REQUESTED** [review]
I feel like a proper perl CPAN module might have been a better choice here.

test fail to find the module still:

```
[16:32:55] t/27-websockets.t ......................................... Can't locate OpenQA/Test/TimeLimit.pm in @INC (you may need to install the OpenQA::Test::TimeLimit module) (@INC contains: /home/squamata/project/t/lib ../lib lib t/lib /home/squamata/project/lib /usr/lib/perl5/site_perl/5.26.1/x86_64-linux-thread-multi /usr/lib/perl5/site_perl/5.26.1 /usr/lib/perl5/vendor_perl/5.26.1/x86_64-linux-thread-multi /usr/lib/perl5/vendor_perl/5.26.1 /usr/lib/perl5/5.26.1/x86_64-linux-thread-multi /usr/lib/perl5/5.26.1 /usr/lib/perl5/site_perl) at t/27-websockets.t line 24.
BEGIN failed--compilation aborted at t/27-websockets.t line 24.
[16:32:55] t/27-websockets.t .........................................   Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Review by okurz — APPROVED** [review]

**Comment by Martchus** [comment]
The main openQA package builds succeeded on OBS - that should be good enough. I'll force push the previous version of the commits because mergify should not have interfered here. (Unfortunately subrepo is referencing Git commit hashes so I believe we must not change the parent commit.)

**Pull request merged**